### ISSUE TITLE
Pit of Saron issues

### ISSUE THREAD

**Issue by LKArthas** [callmephil/SunWellCore#71]
Instance The Pit of Saron has the missing starting Npcs depending on faction even quest starters for factions (Jaina: Alliance; Sylvannas: Horde) Missing Scourgelord Tyranus Starting Event at the entrace of the Dungeon, and Boss Figthing Starting Event has missing spawns regarding Alliance or Horde Sides, and when Scourgelord Tyranus Dies, missing ending Event are not included
Best Regards

**Comment by talamortis** [comment]
duplicate from #37 ?

**Comment by LKArthas** [comment]
yes a basic duplicated, but this reports more issues on the instance ;)
